tests/databrowser.js

```diff
@@ -583,6 +583,23 @@ define([
                 });
             });
 
+            it("should drag/drop case index", function (done) {
+                util.loadXML("");
+                var mug = util.addQuestion("DataBindOnly", "mug"),
+                    input = $("[name=property-calculateAttr]"),
+                    editor = input.ckeditor().editor,
+                    widget = util.getWidget('property-calculateAttr');
+                widget.input.promise.then(function () {
+                    editor.on('change', function() {
+                        assert.equal(mug.p.calculateAttr,
+                            "instance('commcaresession')/session/data/case_id");
```

Comment:
That's correct. I did it this way because `#case/parent` and `#case/grandparent` are ambiguous if we allow that.